chain/arweave/src/adapter.rs

```diff
@@ -40,12 +44,14 @@ impl bc::TriggerFilter<Chain> for TriggerFilter {
 /// see docs: https://thegraph.com/docs/en/supported-networks/arweave/
 #[derive(Clone, Debug, Default)]
 pub(crate) struct ArweaveTransactionFilter {
-    owners: HashSet<Vec<u8>>,
+    owners_pubkey: HashSet<Vec<u8>>,
+    owners_hex: HashSet<Vec<u8>>,
+    match_all: bool,
 }
 
 impl ArweaveTransactionFilter {
     pub fn matches(&self, owner: &[u8]) -> bool {
-        self.owners.contains(owner)
+        self.match_all || self.owners_pubkey.contains(owner) || self.owners_hex.contains(owner)
```

Comment:
`owner` is always in long form here, I don't see how it can actually work.

Reply:
for the time being it will just match on `owners_pubkey` but this means we can switch as soon as we have the short format coming from firehose. Performance is prolly not a concern here yet

Reply:
fixed

Reply:
added some more fixes and some tests as well